data/data/openstack/topology/sg-worker.tf

```diff
@@ -72,7 +72,7 @@ resource "openstack_networking_secgroup_rule_v2" "worker_ingress_heapster_from_m
   security_group_id = openstack_networking_secgroup_v2.worker.id
```

Comment:
Does the worker really have anything listening on port 80 and 443? I believe we can remove these 2 rules.

Reply:
Same comment for `worker_ingress_heapster` and `worker_ingress_heapster_from_master`

Reply:
Noticed AWS doesn't open port 10255 (`worker_ingress_kubelet_secure`). It may not be necessary anymore.

Reply:
Same comment about consistently using the terraform 0.12 syntax.

Reply:
Thanks! I think 80 and 443 are necessary for external worker access (the openshift-router binds to them IIRC).

This is normally proxied by the Ingress VIP (and I'm not sure if the rules are necessary there), but if the deployers want to put up their own LB with direct access to the worker nodes, I think this may be necessary.

I'm going to update the PR with everything else but this and then play with this. It shouldn't be a huge deal to leave those SGs in IMHO.